components/browser/errorpages/src/main/res/raw/error_pages.html

```diff
@@ -26,14 +53,32 @@ <h1 id="et_dnsNotFound" class="errorTitleText">%messageShort%</h1>
 
                 <!-- Short Description -->
                 <div id="errorShortDesc">
-                   %messageLong%
+                    %messageLong%
                 </div>
 
             </div>
 
             <!-- Retry Button -->
             <button id="errorTryAgain" onclick="window.location.reload()">%button%</button>
 
+            <!-- Advanced Button -->
+            <button id="advancedButton" class="buttonSecondary" onclick="toggleAdvanced()" style="display: %advancedSSLStyle%;">Advanced...</button>
+
+            <div id="advancedPanelContainer">
+                <div id="badCertAdvancedPanel" class="advanced-panel" style="display: none;">
+                    <p id="badCertTechnicalInfo">
+                        <label>Someone could be trying to impersonate the site and you should not continue.</label>
```

Comment:
I think this text needs to be in string ressources so that they can be translated, but I'm not sure myself how to do that in the raw HTML without passing them through the Android system (like we've done with the error messages).